.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+name: build
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches: ['main']
+    tags: ['v*']
+
+permissions:
+  contents: read
+
+jobs:
+  BuildCross:
```

Comment:
If possible I would like to stick with `dash-case` (also known as `kebab-case`) naming convention.

Reply:
done